adapters/amqp-vertx/src/test/java/org/eclipse/hono/adapter/amqp/VertxBasedAmqpProtocolAdapterTest.java

```diff
@@ -184,25 +197,52 @@ public void uploadTelemetryMessageWithUnsettledDeliverySemantics() {
         final MessageSender telemetrySender = givenATelemetrySenderForAnyTenant();
 
         // which is enabled for a tenant
-        givenAConfiguredTenant("enabled-tenant", true);
+        givenAConfiguredTenant(TEST_TENANT_ID, true);
 
         // IF a device send telemetry data (with un-settled delivery)
         final ProtonDelivery delivery = mock(ProtonDelivery.class);
         when(delivery.remotelySettled()).thenReturn(false);
-        final ResourceIdentifier resource = ResourceIdentifier.from("telemetry", "enabled-tenant", "the-device");
+        final ResourceIdentifier resource = ResourceIdentifier.from("telemetry", TEST_TENANT_ID, TEST_DEVICE);
```

Comment:
I would also use TelemetryConstants.ENDPOINT_NAME instead of the literal `"telemetry"`